spec/factories/miq_group.rb

```diff
@@ -1,7 +1,8 @@
 FactoryGirl.define do
   factory :miq_group do
+    guid { MiqUUID.new_guid }
+
     sequence(:description) { |n| "Test Group #{seq_padded_for_sorting(n)}" }
-    sequence(:guid) { |n| UUIDTools::UUID.parse_int((0x87654321 << 96) + n) }
```

Comment:
@matthewd this looks good to me.  Note, this changes the change in e0d20e6c447c0840d685e58c552fa73095fe9709 from something I can't easily understand to something more obvious to me.  I don't understand the prior code to feel confident.  Are you ok with this change?